### PR TITLE
REF: navigation styles

### PR DESCRIPTION
**Pull request by limpbrains**
Redoing #2246 

It is quite a big change and can probably break things
I've also refactored imports a little - external modules at first, then internal, I hope you don't mind

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4qccu5aq6rveqhnyvwb0t4ggx4

[download apk](http://filestorage.bluewallet.io:1488/1607103667.apk)

**Comment by Overtorment** [comment]
is this redy for review & merge?

**Review by Overtorment — APPROVED** [review]
i glanced, lgtm, but I know nothing about react navigation

**Comment by limpbrains** [comment]
> is this ready for review & merge?

Yes it does

**Review by marcosrdz — CHANGES_REQUESTED** [review]
LNDViewinvoice is the only screen i seem to find missing the change..or at least its not showing the proper options such as the no back title, back arrow colors and title.

**Comment by limpbrains** [comment]
fixed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ftzb69tf283mw61ag7ycr0dm5m

[download apk](http://filestorage.bluewallet.io:1488/1607150930.apk)

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/hamjw0kav4qppf2mwdzbx2afxm

[download apk](http://filestorage.bluewallet.io:1488/1607410513.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7w963xx9ec70ea1brmgyrzxdd0

[download apk](http://filestorage.bluewallet.io:1488/1607416203.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zbu1q3c00nrzfgk8juc4jzcmwm

[download apk](http://filestorage.bluewallet.io:1488/1607850158.apk)

File: screen/wallets/addMultisigHelp.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import { Image, View, Text, ScrollView, StyleSheet } from 'react-native';
 import { useTheme } from '@react-navigation/native';
 import { SafeBlueArea, BlueNavigationStyle, BlueLoading } from '../../BlueComponents';
```
Comment (Overtorment) [inline screen/wallets/addMultisigHelp.js]:
```
screen/wallets/addMultisigHelp.js#L4
'BlueNavigationStyle' is defined but never used
```

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1ejnxq9fwhc1eg2rqhyv4yb338

[download apk](http://filestorage.bluewallet.io:1488/1607965776.apk)

**Comment by Overtorment** [comment]
is it ready for merge?
also, mind the conflicts

**Comment by Overtorment** [comment]
if its a risky change, lets distribute the build via testflight and give it some proper testing

**Comment by limpbrains** [comment]
I believe it is ready. I will test it once again tomorrow and let's merge!

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wnqcrkrdaknnn9kd19whmvrd7c

[download apk](http://filestorage.bluewallet.io:1488/1608150875.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ep4ad9ucwu73mn7k2bd8y83jvw

[download apk](http://filestorage.bluewallet.io:1488/1608215226.apk)

**Review by Overtorment — APPROVED** [review]
lgtm. merge after @ncoelho blessing

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dgykkznkwveckvjq767h4c6v14

[download apk](http://filestorage.bluewallet.io:1488/1608361759.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kwuw1bqgd2r4537ujgt6dwnh6w

[download apk](http://filestorage.bluewallet.io:1488/1608364750.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/z67yzg99yzy88njz3ba68z84nm

[download apk](http://filestorage.bluewallet.io:1488/1608367806.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wchbz971br6ct3qfam5a9fxdr4

[download apk](http://filestorage.bluewallet.io:1488/1608371892.apk)

**Comment by limpbrains** [comment]
There were problems with e2e tests. Now they are passing. But now regular tests are failing. I don't know why. I think the PR is ready to be merged

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/89t3cngquk2wmv1udtb0nx1dp4

[download apk](http://filestorage.bluewallet.io:1488/1608711608.apk)

**Comment by marcosrdz** [comment]
Conflicts

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gy4qc7jghh4xz0emjxhz2nu9ec

[download apk](http://filestorage.bluewallet.io:1488/1608843150.apk)

**Comment by marcosrdz** [comment]
lol conflicts again

**Comment by limpbrains** [comment]
> lol conflicts again

yeah, it was a bad idea to resort imports in this PR

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dgfppkfufeb97ra62x3ub1je7c

[download apk](http://filestorage.bluewallet.io:1488/1608879847.apk)

**Comment by Overtorment** [comment]
@ncoelho

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/r3zvw3av0jn6jz81mzg40j8168

[download apk](http://filestorage.bluewallet.io:1488/1608913934.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bvjv8k1547g1emd05zahn8x598

[download apk](http://filestorage.bluewallet.io:1488/1609665445.apk)

**Comment by limpbrains** [comment]
Let's merge it

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/7cr3ebwm0dv20w63qw7e7f6g68

[download apk](http://filestorage.bluewallet.io:1488/1609831790.apk)

**Comment by ncoelho** [comment]
Screenshot 2021-01-05 at 13 39 31

**Comment by Overtorment** [comment]
@ncoelho if you also submit review with "changes requested", PR developer will have a chance to fix issues and then re-request a review and you will receive a notification about it

File: screen/wallets/viewEditMultisigCosigners.js [hunk 2]
```diff
@@ -633,9 +633,9 @@ const styles = StyleSheet.create({
   },
 });
 
-ViewEditMultisigCosigners.navigationOptions = ({ navigation }) => ({
-  ...BlueNavigationStyle(navigation, true),
-  title: loc.multisig.manage_keys,
+ViewEditMultisigCosigners.navigationOptions = navigationStyle({
+  closeButton: true,
+  title: loc.multisig.view_edit_cosigners_title,
```
Comment (ncoelho) [inline screen/wallets/viewEditMultisigCosigners.js:638]: Title not reflected

**Comment by limpbrains** [comment]
@ncoelho 
thanks, fixed!

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9r1dgm4nhrwgyy0dkgtuh18w8r

[download apk](http://filestorage.bluewallet.io:1488/1609875501.apk)

**Review by ncoelho — APPROVED** [review]

**Pull request merged**